src/rpc/mission.rs

```diff
@@ -44,11 +44,19 @@ impl MissionService for MissionRpc {
         &self,
         request: Request<mission::v0::StreamUnitsRequest>,
     ) -> Result<Response<Self::StreamUnitsStream>, Status> {
+        let request_message = request.into_inner();
+
+        if request_message.category == 0 {
+            return Err(Status::out_of_range(
+                "Specify the category of the unit to stream",
+            ));
+        }
```

Comment:
Why not keep support for allowing a stream that includes all unit types - I'd assume not everyone wants to stream different unit types at different update rates?

Reply:
+1 The clients I am writing don't need this granularity.

Question for @rkusa Given https://github.com/DCS-gRPC/rust-server/issues/20 is still open. Does having potentially many streams active for each client (vs the one-per-client now), like this feature opens up, have any performance implications we need to actually worry about.

Reply:
@rurounijones assuming that the streams observe different units, there is almost no additional work done compared to observing all in one stream.

Reply:
I've now implemented `UNSPECIFIED` would monitor all units. If someone desires to have different polling variations, they are welcome to join the observation streams on the client app.